gisnav:
  mock_gps_node:
    ros__parameters: []
